server/backend/database/mongo/client.go

```diff
@@ -1149,62 +1149,170 @@ func (c *Client) UpdateAndFindMinSyncedTicket(
 	), nil
 }
 
-// UpdateAndFindMinSyncedVersionVector updates the given serverSeq of the given client
-// and returns the SyncedVersionVector of the document.
-func (c *Client) UpdateAndFindMinSyncedVersionVector(
+// UpdateAndFindMinSyncedVersionVectorAfterPushPull returns min synced version vector
+func (c *Client) UpdateAndFindMinSyncedVersionVectorAfterPushPull(
```

Comment:
Since we are not using MongoDB Transaction, we need to handle exceptions for partial failures. I will request a meeting about this later.

Reply:
I wrote new TC about this case https://github.com/yorkie-team/yorkie/pull/981/commits/ada30e04fd01ccad8cb50b11b90389b85ae9e76a
Assume there're three clients, and one detached but its version vector remains in DB. (partial failure)

The result was same as we expected, no GC runs because min version vector stays the same as when one client detached. Except this, others work properly.

If this case really happened, I think housekeeping should handle this situation